docs/standard/categories-list.rst

```diff
@@ -69,6 +69,7 @@ Valid Tags
 - it-service-management
 - knowledge-management
 - learning-management-system
+- library-management-system
```

Comment:
```suggestion
- library
```

Reply:
This can potentially be confused with a software library.
What about `integrated-library-system`? https://en.wikipedia.org/wiki/Integrated_library_system

Reply:
yes!